src/frontend/components/UI/Sidebar/index.tsx

```diff
@@ -8,7 +8,7 @@ import { DMQueueElement } from 'common/types'
 
 import { ReactComponent as HeroicIcon } from 'frontend/assets/heroic-icon.svg'
 
-let sidebarSize = localStorage.getItem('sidebar-width') || 240
+let sidebarSize = Number(localStorage.getItem('sidebar-width')) || 240
```

Comment:
Not sure if this is correct here. Node's type definitions say that `getItem` only ever returns a string, but we want a number here